### ISSUE TITLE
Any initialization tech in SSNorm?

### ISSUE THREAD

**Issue by yuyangxie96** [dmis-lab/Outlier-Safe-Pre-Training#5]
I'm trying to reproduce your pretraining experiment. However, I've noticed that even with our model trained on 1TB of data, we are still observing massive activations in the input of down_proj layer. Additionally, the values in our normalization layers are significantly larger compared to those in the checkpoint you provided. I was wondering if there are any special techniques for initializing the normalization layers?
